### ISSUE TITLE
GGClassic - issues with dec 06 2022 update

### ISSUE THREAD

**Issue by ssatguru** [TheGameCreators/GameGuruRepo#4010]
I am having issues after the latest update. wonder if other people are experiencing the same too.

For example when trying to load and test the "death-valley" map here is what I saw
clicking "test game" took about 42 secs for the game to load
fps fell down to 24 fps
when i picked up a gun it did not show up. I eventually found that if I slid "Lightrays" to 0 under Graphic Options it did show up
fps hovered around 21 fps making the game completely unplayable.

I went to steam and rolled back to sept-2022 update
(if anybody interested , instructions on rolling back - https://steamcommunity.com/app/266310/discussions/0/1318835718950057499/)
clicking "test game" took about 20 secs for the game to load
fps started with 45 fps and then hovered around 35 fps
No issues picking up gun or playing the game.

went back still further to last year dec-2021 update
clicking "test game" took about 23 secs for the game to load
fps started with 60 fps and then hovered around 40 fps
No issues picking up gun or playing the game.

**Comment by cybernescence** [comment]
I've fixed the weapon issue, will need a TGC review and to be released into the steam build at some point. For now don't use the light rays (slider to zero).
With regard to the FPS drops - there have been a number of updates that may do this, depending on your PC specs - the most likely is the new HBAO - try dropping the 'SAO Intensity' slider to zero - if this then recovers your FPS then you can revert to the old SSAO method.
I've no idea why your load times have increased.
Cheers

**Comment by ssatguru** [comment]
@cybernescence  Thanks for looking into this.

Also my apologies. On re-reading  my comments I realized I had made a mistake. I have since re-edited and corrected my comment.
The correction was to state that I did try the previous update namely the "Sept, 2022" update and did not find any issues.
For now I have reverted back to the Sept, 2022 update and will continue to work with that.

For those who may not be familiar, switching updates is pretty straightforward in Steam. I was afraid it would try to uninstall the the current version and reinstall the full previous version resulting in gigabytes of downloads. Surprisingly it does not do so. Just few MBs downlaod, few minutes and it is done, :)

Anyway here are the specs for my computer if it helps.
OS : WIndows 10 Home, 22H2
CPU : AMD64 , FX-8350 EIGHT CORE, 4 GHz
MEMORY: 32GB - 4 8GB DDR3 667MHZ 
GPU: NVIDIAS GeForce GTX 1060 6GB 
HD:1TB SAMSUNG EVO SSD
